schema.sql

```diff
@@ -6,6 +6,7 @@ CREATE TABLE animals (
     date_of_birth date,
     escape_attempts int,
     neutred boolean,
-    weight_kg DECIMAL(10,2)
+    weight_kg DECIMAL(10,2),
+    species varchar(100)
```

Comment:
Kindly modify the animals table by using the `ALTER TABLE` statement instead of manually editing the create table statement. Please note that it may not always be possible to simply alter a table by changing the table creation method as this can lead to a lot of conflict especially when dealing with tables that store large amounts of data👍